src/v/redpanda/application.h

```diff
@@ -212,6 +215,11 @@ class application {
     }
 
 private:
+    // First warning timeout
+    static constexpr auto short_shutdown_warning_timeout = 30s;
+    // Time after which we will print the warning about the service shutdown
+    // taking too long.
+    static constexpr auto long_shutdown_warning_timeout = 180s;
```

Comment:
just curious about these values. is this way too long? for example, in ducktape tests don't we only give redpanda a total of like 30 seconds to shutdown _everything_?

Reply:
so maybe short timeout should be 10s and long 60s, WDYT?

Reply:
i'd grab snag a few debug and release logs from a CI run and a CDT run and do a quick grep to see what kind of times make sense for the current system.

also, if we make these tunable configuraiton options then we can think less about the initial values they take on.

Reply:
the hang that I observed in CDT was around 6 minutes before RP was sigkilled

Reply:
I tweaked the timeouts a bit. Now they're 15s and 120s.

Reply:
>the hang that I observed in CDT was around 6 minutes before RP was sigkilled

but we aren't interested in this case, right? we are interested in: what are the normal threshold values?